R/initialize.R

```diff
@@ -21,9 +21,13 @@ sd_initialize <- function(self, private, path, loadTimeout, checkNames,
   self$logEvent("Getting PhantomJS port")
   private$phantomPort <- get_phantomPort(timeout = phantomTimeout)
 
-  if (grepl("^http(s?)://", path)) {
-    private$setShinyUrl(path)
+  if(!dir_exists(path))
```

Comment:
Can you please follow https://style.tidyverse.org here? 

* Space after `if`
* `{}` around stop
* `<-` instead of `=`